tests/cbmc/proofs/s2n_dh_p_g_Ys_to_dh_params/Makefile

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Expected runtime is 7 seconds.
+
+MAX_BLOB_SIZE = 10
```

Comment:
Maybe add a comment about why this size was chosen? Even if it's just a simple "was good for performance and didn't hurt coverage", it might be future proof writers.

Reply:
I'll address this comment on https://github.com/awslabs/s2n/pull/2433.